appengine/pond/duck/js/duck.js

```diff
@@ -60,21 +69,15 @@ Pond.Duck.init = function () {
       if (Blockly.utils.dom.hasClass(tabs[selectedIndex], 'tab-disabled')) {
         return;
       }
-      for (var i = 0; i < tabs.length; i++) {
-        if (selectedIndex == i) {
-          Blockly.utils.dom.addClass(tabs[i], 'tab-selected');
-        } else {
-          Blockly.utils.dom.removeClass(tabs[i], 'tab-selected');
-        }
-      }
+      Pond.Duck.selectTab(tabs, selectedIndex);
       Pond.Duck.changeTab(selectedIndex);
     };
   }
```

Comment:
What I meant was to factor out this whole method. It doesn't look like there's a case where you wouldn't also want to call changeTab.

If you create a method that takes selectedIndex as a parameter that updates the tab classes and calls changeTab, then you can call it in Pond.Duck.Online.updateEditor and you can bind it to the button with the correct paramter using: `BlocklyGames.bindClick(tabs[i], function() {Pond.Duck.selectTab(i)}`

Reply:
Just found out the anonymous function doesn't work, what does work is 
`Pond.Duck.selectTab.bind(this, i)`
Although I commented about it, I'm going to end up modifying things anyways when I add the new tab, so I'm cool with just leaving the refactor of selectTab as is for this PR.